### ISSUE TITLE
2.4.2 Bed keeps disappearing and also PotionPlus Zoe's alchemy table and Potion Cauldron

### ISSUE THREAD

**Issue by Neoony** [zolantris/ValheimMods#164]
### Affected Mod

ValheimRaft

### What happened? 🤔

Its a bit strange, because the bed was fine for quite a while after the claiming/respawning on bed was fixed. (I guess after I went to 2.4.1 or 2.4.2)

But then since some time, the bed just keeps disappearing when I leave the area.
Sometimes it randomly reappears on the same spot.
The spawning on that place usually still works, but just noticed that sometimes in rare cases it puts me somewhere in the ocean where I was at some point.

The other thing is
PotionPlus https://thunderstore.io/c/valheim/p/OdinPlus/PotionPlus/

It used to be that when I placed the Zoe's Alchemy Table, the table would float and go up and down independently from the ship. (and it kept going much higher sometimes randomly...sometimes I saw it on second floor of the ship)
But now, since around the same time as the bed got fixed, the table and the cauldron just disappears in the same way as the bed. And sometimes randomly reappears. Or sometimes randomly you can see it quite a bit outside of the ship.

![valheim_2024-12-06_18-36-01](https://github.com/user-attachments/assets/c5b1e065-4fae-44a3-898d-5a4f630b0955)

![valheim_2024-12-06_18-34-41](https://github.com/user-attachments/assets/624aa755-cde8-4a65-882c-81fec56ffea4)

![valheim_2024-12-06_21-54-14](https://github.com/user-attachments/assets/dc2b2e2f-577c-4d51-8bd7-cfa7e7f3b385)

![valheim_2024-12-06_21-57-02](https://github.com/user-attachments/assets/0ed0c2de-b2a5-4f9c-a253-99d0578fbf27)

Also, not completely sure
But sometimes randomly my boat just decides to sink to the ocean floor
And then I have to do some vehicle creative and rotate on it and get on the steering and just do all that few times and it fixes itself.

And I am not sure, but it might also be related to the fact that I sometimes place new bed and alchemy table (without removing the ones disappearing, because they are not there...and I need the comfort, or to craft the potions)
And perhaps something then keeps building up and making the ship sink.

Sometimes the old disappearing bed/table would just reappear in the same place and then I would have double in the same spot (until it would disappear again)

Maybe its related to my ship sinking more often than others, maybe not.

### Steps to reproduce 🔄

Place a bed and leave the area or move the ship a bit and then leave area
Same thing with the alchemy table / potion cauldron
Not sure what exactly triggers it, but mostly (but not always) when you leave area and come back

### Log Outputs 📜

```txt
Log is too long
```

### Environment 🌍

```markdown
OS:
- **OS**: Windows 10 Pro 64bit
- **Graphics**: DX11
Mods:
- **BepInEx**: 5.4.2202
- **Jotunn**: 2.21.3
- **ValheimRAFT**: 2.4.2 (I am not on 2.4.3 because of the stutter and weird steering [see other issue] )
Advize-PlantEasily
Advize-PlantEverything
ASharpPen-Custom_Raids
ASharpPen-This_Goes_Here
Azumatt-AAA_Crafting
Azumatt-AzuAreaRepair
Azumatt-AzuClock
Azumatt-AzuContainerSizes
Azumatt-AzuCraftyBoxes
Azumatt-AzuExtendedPlayerInventory
Azumatt-AzuMiscPatches
Azumatt-AzuWearNTearPatches
Azumatt-AzuWorkbenchTweaks
Azumatt-FirstPersonMode
Azumatt-ILoxYou
Azumatt-ImFRIENDLY_DAMMIT
Azumatt-MistrootTamer
Azumatt-Official_BepInEx_ConfigurationManager
Azumatt-PerfectPlacement
Azumatt-Queue_Me_Maybe
Azumatt-SearchableBuildMenu
Azumatt-SleepSkip
BadgersValheimSkies
BentoG-MissingPieces
BetterUI_ForeverMaintained-BetterUI_ForeverMaintained
blacks7ar-CrystalCollector
blacks7ar-GuckCollector
blacks7ar-Hunting
blacks7ar-ResinCollector
blacks7ar-SeedBed
blacks7ar-SimpleElevators
blacks7ar-SNEAKer
blacks7ar-TarCollector
blacks7ar-VikingsDoSwim
blacks7ar-WeaponHolsterOverhaul
coemt-Buoys
ComfyMods-Chatter
ComfyMods-EulersRuler
CookieMilk-MajesticChickens
CustomAudio
CW_Jesse-BetterNetworking_Valheim
Digitalroot-Digitalroots_Slope_Combat_Assistance
Digitalroot-Heightmap_Unlimited_Remake
Digitalroot-Max_Dungeon_Rooms
Digitalroot-Triple_Bronze_JVL
FixItFelix-AutoMapPins
FixItFelix-DiscoveryRadius
Goldenrevolver-Quick_Stack_Store_Sort_Trash_Restock
HDPortalFXData
HDTerrainData
hyleanlegend-Rune_Magic
ishid4-BetterArchery
jcdcdev-Signs
JereKuusela-Server_devcommands
JereKuusela-World_Edit_Commands
JewelHeim-EpicLoot_Balanced
JewelHeim-EpicLoot_Therzie
JewelHeim-Monstrum_Raids
JewelHeim-Wizardry_Raids
KGvalheim-No_More_Crashes
Korppis-ReliableBlock
LVH-IT-UseEquipmentInWater
MaxFoxGaming-Coal_Control
MaxFoxGaming-Procreation_Plus
MSchmoecker-ContentsWithin
MSchmoecker-ItemHopper
MSchmoecker-MultiUserChest
MSchmoecker-PressurePlate
MSchmoecker-TimeoutLimit
MSchmoecker-VNEI
Neobotics-StopChoppyAudio
oathorse-DudeWheresMyPortal
OdinPlus-BetterLanterns
OdinPlus-CraftyCartsRemake
OdinPlus-OdinCampsite
OdinPlus-OdinHorse
OdinPlus-OdinsFoodBarrels
OdinPlus-OdinsKingdom
OdinPlus-OdinTrainingPlace
OdinPlus-PotionPlus
OdinPlus-QuickTeleport
OdinPlus-TeleportEverything
RandyKnapp-EpicLoot
RandyKnapp-EquipmentAndQuickSlots
RandyKnapp-MinimalStatusEffects
Riintouge-MagneticWishbone
RustyMods-Almanac
RustyMods-FishTrap
Searica-AdvancedTerrainModifiers
Searica-CameraTweaks
Searica-SafetyStatus
Searica-ToolTweaks
shudnal-BuyDvergrExtractorNeedle
shudnal-CircletExtended
shudnal-Compass
shudnal-Firefly
shudnal-HipLantern
shudnal-LongshipUpgrades
shudnal-StaminaExtended
sinai-dev-UnityExplorer
Smoothbrain-Blacksmithing
Smoothbrain-Building
Smoothbrain-CombatOwner
Smoothbrain-ComfortTweaks
Smoothbrain-Cooking
Smoothbrain-CreatureLevelAndLootControl
Smoothbrain-DualWield
Smoothbrain-Evasion
Smoothbrain-Exploration
Smoothbrain-Farming
Smoothbrain-Foraging
Smoothbrain-Lumberjacking
Smoothbrain-Mining
Smoothbrain-PassivePowers
Smoothbrain-Ranching
Smoothbrain-Sailing
Smoothbrain-SailingSpeed
Smoothbrain-ServerCharacters
Smoothbrain-SmoothSave
Smoothbrain-SteadyRegeneration
Smoothbrain-Tenacity
SpikeHimself-XPortal
TastyChickenLegs-BlastFurnaceTakesAll
TastyChickenLegs-LongerDays
TastyChickenLegs-NoSmokeStayLit
Therzie-Armory
Therzie-Monstrum
Therzie-Warfare
Therzie-Wizardry
ValheimModding-Jotunn
Vapok-AdventureBackpacks
VentureValheim-Venture_Floating_Items
virtuaCode-EquipWheel
zolantris-ValheimRAFT
```

**Comment by zolantris** [comment]
Duplication sounds like a mod integration problem with valheim raft. Raft does not create new prefabs so something is wrong. 

Never seen beds duplicated locally guessing a mod causing this too. Beds keep their placement position but otherwise should be safe. Maybe there's a problem with the distance. Ie it's far away from the initial raft object and some players have different loading distances set (or mods that set it).

**Comment by Neoony** [comment]
No, I meant that I would place a second bed after it disappeared. And then later the old one would sometimes reappear in the same place.
Same for the Alchemy table / cauldron.
The old one would just sometimes reappear, whether I placed another one or not.

Nothing is duplicating.

With the alchemy table it happens usually as soon as you leave the area or move the boat. No other players involved.
Clearly the raft mod has an issue with this PotionPlus mod. (while around 130 other mods are fine)
It just disappears or behaves strangely.

The bed is still good so far since I removed any alchemy table / cauldron.

**Comment by Neoony** [comment]
And today suddenly one of my old beds reappears xD

I thought I already removed all of them and placed the Dragon bed, which has disappeared few days ago after being fine for a long time.

But I guess I still had the basic bed there :D

![valheim_2024-12-21_01-27-25](https://github.com/user-attachments/assets/83db20fa-f1a0-4cdc-ad10-76d0ec2d112b)

**Comment by zolantris** [comment]
Closing this. Please reopen if it continues. I'm guessing this won't occur on a new vehicle but maybe it's the mod list causing a conflict.
